src/watcher.js

```diff
@@ -135,6 +138,9 @@ class Watcher extends SafeEventEmitter {
                 case routes.DIRECTORY_FOLLOWING:
                     this.waitForLoad('following').then(() => this.emit('load.directory.following'));
                     break;
+                case routes.DIRECTORY_FOLLOWING_LIVE:
+                    this.waitForLoad('followingLive').then(() => this.emit('load.directory.following.live'));
```

Comment:
why does this only target the following pages? are vodcasts on the main directory pages?

Reply:
I've only seen vodcasts on following pages and only in the Overview and Channels tab.